### PR TITLE
Feature/watch list page

### PR DESCRIPTION
**Pull request by ioanagirbap33**

### CODE REVIEW

File: frontend/src/components/card/Card.styled.tsx [hunk 1]
```diff
@@ -3,20 +3,20 @@ import { SPACE_CADET, FRENCH_MAUVE } from "../../Variables";
...
 interface StyledCardProps {
-  variant?: "collection"; // Definiți tipurile acceptate pentru proprietatea variant
+  variant?: "collection";
 }
 
 export const StyledCard = styled.div<StyledCardProps>`
   display: flex;
   align-items: center;
-  justify-content: center;
+  // justify-content: center;
```
Comment (roxanap33) [inline frontend/src/components/card/Card.styled.tsx]: Delete this comment

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import { rem } from "../../utils";
+
+export const MovieCardWatchList = styled.div`
+  display: flex;
+  gap: ${rem(30)};
+  justify-content: space-between;
+  width: 100%;
+  @media (max-width: 450px) {
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx:10]: Add a new line before each media query for better visibility

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  display: flex;
+  gap: ${rem(20)};
+  @media (max-width: 450px) {
+    gap: ${rem(5)};
+  }
+`;
+
+export const MovieDetailsTitle = styled.div`
+  font-size: ${rem(24)};
+  @media (max-width: 450px) {
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx:32]: Here too

File: frontend/src/components/card/Card.styled.tsx (hunk 1 above)
Comment (adrianacostep33) [inline frontend/src/components/card/Card.styled.tsx]: Remove this unused line

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  }
+`;
+export const MovieDetailsDescription = styled.div`
+  font-size: ${rem(14)};
+`;
+export const MovieDetailsRight = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  //   align-items: right;
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx]: Delete this

File: frontend/src/components/movie-card-content/MovieCardContent.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <div>
+            <MovieDetailsTitle>{title}</MovieDetailsTitle>
+            <MovieDetailsDescription>{description}</MovieDetailsDescription>
+          </div>
+        </MovieDetailsLeft>
+
+        <MovieDetailsRight>
+          <Rating>
+            <StarIcon sx={{ color: FRENCH_MAUVE }} />
+
```
Comment (roxanap33) [inline frontend/src/components/movie-card-content/MovieCardContent.tsx:43]: Delete this empty line

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx (hunk 2 above)
Comment (adrianacostep33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx:10]: Add a line before media query

File: frontend/src/pages/watch-list/WatchListPage.styled.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import { rem } from "../../utils";
+
+export const WatchListContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${rem(20)};
+  margin: ${rem(0)} ${rem(100)} ${rem(100)} ${rem(200)};
+  padding-top: ${rem(150)};
+  @media (max-width: 800px) {
```
Comment (roxanap33) [inline frontend/src/pages/watch-list/WatchListPage.styled.tsx:11]: New line here too

File: frontend/src/pages/watch-list/WatchListPage.styled.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+export const WatchListTitle = styled.div`
+  font-size: ${rem(48)};
+`;
+
+export const WatchListMovieCollectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${rem(50)};
+  @media (max-height: 500px) {
```
Comment (roxanap33) [inline frontend/src/pages/watch-list/WatchListPage.styled.tsx:25]: New line

File: frontend/src/components/movie-card-content/MovieCardContent.styled.tsx (hunk 4 above)
Comment (adrianacostep33) [inline frontend/src/components/movie-card-content/MovieCardContent.styled.tsx]: Delete this line

File: frontend/src/pages/watch-list/WatchListPage.tsx [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+  description: string;
+  imagesUrls: string[];
+}
+
 const WatchListPage = () => {
-  return <div></div>;
+  const { loading, error, data } = useQuery(GET_MOVIES);
+  console.log({ data });
+
+  // if (!data)
```
Comment (roxanap33) [inline frontend/src/pages/watch-list/WatchListPage.tsx]: Delete this

File: frontend/src/pages/watch-list/WatchListPage.tsx [hunk 9]
```diff
@@ -1,5 +1,59 @@
...
+    <>
+      <WatchListContainer>
+        <WatchListTitle>WATCHLIST</WatchListTitle>
+        <WatchListMovieCollectionContainer>
+          {data &&
+            data.userQuery.user.watchedList.map((movie: IMovie) => (
+              <MovieCardContent
+                key={movie.id}
+                picture={movie.imagesUrls[1]}
+                // picture={""}
```
Comment (roxanap33) [inline frontend/src/pages/watch-list/WatchListPage.tsx]: Delete this

File: frontend/src/pages/home/HomePage.tsx [hunk 10]
```diff
@@ -19,14 +19,14 @@ interface IMovie {
 }
 
 const HomePage = () => {
-  const { loading, error, data } = useQuery(GET_MOVIES);
+  // const { loading, error, data } = useQuery(GET_MOVIES);
```
Comment (roxanap33) [inline frontend/src/pages/home/HomePage.tsx]: Delete those comments

File: frontend/src/pages/watch-list/WatchListPage.styled.tsx (hunk 6 above)
Comment (adrianacostep33) [inline frontend/src/pages/watch-list/WatchListPage.styled.tsx:11]: Add a line before media query

File: frontend/src/pages/watch-list/WatchListPage.tsx [hunk 11]
```diff
@@ -1,5 +1,59 @@
...
+  imagesUrls: string[];
+}
+
 const WatchListPage = () => {
-  return <div></div>;
+  const { loading, error, data } = useQuery(GET_MOVIES);
+  console.log({ data });
+
+  // if (!data)
+  // data.movieQuery.movies.map((movie: IMovie) => console.log(movie));
```
Comment (adrianacostep33) [inline frontend/src/pages/watch-list/WatchListPage.tsx]: Delete this commented line

File: frontend/src/pages/watch-list/WatchListPage.tsx (hunk 9 above)
Comment (adrianacostep33) [inline frontend/src/pages/watch-list/WatchListPage.tsx]: Do we need this?

**Review by roxanap33 — APPROVED** [review]

**Review by adrianacostep33 — APPROVED** [review]

**Pull request merged**